const.go

```diff
@@ -45,6 +45,8 @@ const (
 	clientSecureConn
 	clientMultiStatements
 	clientMultiResults
+	clientPSMultiResults
```

Comment:
Why the `clientPSMultiResults`? Is it necessary for this PR?

Reading the description at https://dev.mysql.com/doc/internals/en/capability-flags.html#flag-CLIENT_PS_MULTI_RESULTS it looks like we might have forgotten. If this is the case, please open a separate PR for this.